app/src/main/java/org/fossasia/pslab/fragment/SavedExperiments.java

```diff
@@ -778,4 +827,19 @@ private void DisplayTemporaryToast() {
         Toast.makeText(getActivity(),getString(R.string.temporary_toast),Toast.LENGTH_SHORT).show();
     }
 
+    private ArrayList<SearchModel> initData() {
+        ArrayList<SearchModel> items = new ArrayList<>();
+        for (String anExperimentsArray : experimentsArray) {
+            items.add(new SearchModel(anExperimentsArray));
+        }
+        return items;
+    }
+
+    private void runExperiment(int location){
```

Comment:
Run a code reformat to this file as well to fix all the indentation and spaces issues.

Reply:
Sure @CloudyPadmal